src/store/store.ts

```diff
@@ -1,17 +1,21 @@
 import create from 'zustand';
 import { Cookbook } from '../models/Cookbook';
 import { Guide } from '../models/Guide';
+import { SectionModel } from '../models/Section';
 import { User } from '../models/User';
 
 export interface IStore {
   user: User | null;
   cookbook: Cookbook | null;
   cookbooks: Cookbook[];
   guides: Guide[];
+  sections: SectionModel[];
```

Comment:
hmmm not sure if we need `sections` since they will already be populated in a guide

Reply:
I don't think they should be a separate field but a selector to get it off a cookbook based off an ID could work.

Reply:
think thats kinda how u wanted it with the new change?